Zilliqa

```diff
@@ -0,0 +1,5 @@
+Blockchain Software Engineer (C++)
+Compiler Engineer
+Full-Stack Developer
+Blockchain Application Engineer
```

Comment:
**As a Blockchain Application Engineer, you will:**
- Develop high-throughput decentralized applications on Zilliqa.
- Conduct thorough tests to demonstrate the performance and security of the applications.
- Keep up with latest technologies in the blockchain space and understand when and where to adopt them.
- Support the development of Zilliqa core infrastructure and smart contract.
- Work closely with collaborators in the community or partner enterprises.

**Qualifications**

- Bachelor’s degree (or higher) in computer science, computer engineering, or equivalent practical experience.
- 3+ years hands-on experience in developing large-scale applications.
- Rich experience in development on blockchains such as writing Solidity contracts is a plus.
- Ability to work independently and collaboratively in an agile team.

https://zilliqa.com/careers/blockchain-application-engineer.html